keyboards/linworks/fave84h/config.h

```diff
@@ -14,49 +14,21 @@ GNU General Public License for more details.
 You should have received a copy of the GNU General Public License
 along with this program. If not, see <http://www.gnu.org/licenses/>.
 */
-
 #pragma once
 
 #include "config_common.h"
-
-/* key matrix size */
-#define MATRIX_ROWS 6
-#define MATRIX_COLS 17
-
-/*
- * Keyboard Matrix Assignments
- *
- * Change this to how you wired your keyboard
- * COLS: AVR pins used for columns, left to right
- * ROWS: AVR pins used for rows, top to bottom
- * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
- *         ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
- *
-*/
-
-// Checked with Eagle Schematic
-#define MATRIX_ROW_PINS { B1, B2, B3, D3, D1, D0  }
-#define MATRIX_COL_PINS { F0, F1, F4, F5, F6, F7, C7, C6, B6, B5, B4, D7, D6, D4, B0, B7, E6}
-
-/* COL2ROW or ROW2COL */
-#define DIODE_DIRECTION COL2ROW
-
-/* Debounce reduces chatter (unintended double-presses) - set 0 if debouncing is not needed */
-#define DEBOUNCE 5
-
 /* Define RGB */
-#define RGBLED_NUM 86
-#define RGB_MATRIX_LED_COUNT 86
-#define RGB_DI_PIN D2
-
 #ifdef RGB_MATRIX_ENABLE
 
+#define RGB_MATRIX_LED_COUNT 126
+#define RGB_DI_PIN D2
+
 #define RGB_DISABLE_WHEN_USB_SUSPENDED
 #define RGB_MATRIX_KEYPRESSES 
 #define RGB_MATRIX_FRAMEBUFFER_EFFECTS
 #define RGB_MATRIX_MAXIMUM_BRIGHTNESS 120
 
-#define RGB_MATRIX_DEFAULT_MODE RGB_MATRIX_CYCLE_ALL
+#define RGB_MATRIX_STARTUP_MODE RGB_MATRIX_CYCLE_ALL
```

Comment:
`RGB_MATRIX_STARTUP_MODE` isnt a thing anymore

```suggestion
#define RGB_MATRIX_DEFAULT_MODE RGB_MATRIX_CYCLE_ALL
```